modules/cert-manager-acme-challenges-types.adoc

```diff
@@ -12,6 +12,13 @@ HTTP-01:: With the HTTP-01 challenge type, you provide a computed key at an HTTP
 +
 For more information, see link:https://cert-manager.io/docs/configuration/acme/http01/[HTTP01] in the upstream cert-manager documentation.
 
+[NOTE]
+====
+HTTP-01 requires _Let's Encrypt_ staging environment to access the cluster's route to issue a certificate. Note that using internal or private clusters that are behind the proxy to access, would fail the HTTP-01 to issue a certificate. 
```

Comment:
```suggestion
HTTP-01 requires _Let's Encrypt_ servers to access the cluster's route. Note that using internal or private clusters behind the proxy to access, would fail the HTTP-01 validation checks required to issue a certificate. 
```

Reply:
Ack.